packages/main/src/TableRow.ts

```diff
@@ -117,8 +117,16 @@ class TableRow extends TableRowBase {
 	get _isInteractive() {
 		return this.interactive;
 	}
+
+	get isTableRow() {
+		return true;
+	}
 }
 
 TableRow.define();
 
+const isInstanceOfTableRow = (obj: any): obj is TableRow => "isTableRow" in obj;
```

Comment:
this should be in the base and there is a method there "isHeaderRow" which returns false by default and headerRow overwrites it, so if that returns false and if the method exists then it is a row